### PR TITLE
Feature/uodate recipe option

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### This PR fixes:

- Set recipe resources (core_number, memory, time) on CLI for specific recipe(s)

### How to test:

- Automatic and continuous test pass

### Expected outcome:
- Installation, unit and integration tests pass

### Review:
- [ ] Code review
- [ ] New code is executed and covered by tests
- [ ] Tests pass

### CODE REVIEW

File: lib/MIP/Set/Parameter.pm [hunk 1]
```diff
@@ -1156,6 +1157,47 @@ sub set_recipe_mode {
...
+## Returns  :
+## Arguments: $active_parameter_href => Holds all set parameter for analysis {REF}
+
+    my ($arg_href) = @_;
+
+    ## Flatten argument(s)
+    my $active_parameter_href;
+
+
+    my $tmpl = {
```
Comment (jemten) [inline lib/MIP/Set/Parameter.pm]: Remove blank line

File: lib/MIP/Set/Parameter.pm [hunk 2]
```diff
@@ -1156,6 +1157,47 @@ sub set_recipe_mode {
     return;
 }
 
+sub set_recipe_resource {
```
Comment (jemten) [inline lib/MIP/Set/Parameter.pm]: This sub needs tidying

File: lib/MIP/Set/Parameter.pm (hunk 1 above)
Comment (jemten) [inline lib/MIP/Set/Parameter.pm]: Extra empty line

File: lib/MIP/Set/Parameter.pm [hunk 3]
```diff
@@ -1156,6 +1157,47 @@ sub set_recipe_mode {
...
+    };
+
+    check( $tmpl, $arg_href, 1 ) or croak q{Could not parse arguments!};
+
+    my %set_hash_key_map = (set_recipe_core_number => q{recipe_core_number},
+    set_recipe_time => q{recipe_time},
+    set_recipe_memory => q{recipe_memory},
+      );
+
+HASH_KEY:
```
Comment (jemten) [inline lib/MIP/Set/Parameter.pm]: Indentation seems of for this section

File: lib/MIP/Set/Parameter.pm [hunk 4]
```diff
@@ -1156,6 +1157,47 @@ sub set_recipe_mode {
...
+            strict_type => 1,
+        },
+    };
+
+    check( $tmpl, $arg_href, 1 ) or croak q{Could not parse arguments!};
+
+    my %set_hash_key_map = (set_recipe_core_number => q{recipe_core_number},
+    set_recipe_time => q{recipe_time},
+    set_recipe_memory => q{recipe_memory},
+      );
```
Comment (jemten) [inline lib/MIP/Set/Parameter.pm]: would prefer to write a hash like this:
```perl
my %set_hash_key_map = (
    set_recipe_core_number => q{recipe_core_number},
    set_recipe_time        => q{recipe_time},
    set_recipe_memory      => q{recipe_memory},
);
```

File: t/set_recipe_resource.t [hunk 5]
```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env perl
```
Comment (jemten) [inline t/set_recipe_resource.t]: I don't think that this sub has been tidied either. 
The indentation is a bit wonky

File: lib/MIP/Set/Parameter.pm (hunk 4 above)
Comment (henrikstranneheim) [reply lib/MIP/Set/Parameter.pm]: tidy will make it so (if you run it that is 😅 )

**Pull request merged**